server/startup.coffee

```diff
@@ -94,6 +96,26 @@ Meteor.startup ->
               id: "userSpecifiedDisease:#{incident.disease}"
               text: "Other Disease: #{incident.disease}"
 
+  promedFeed = Feeds.findOne(url: $regex: /promedmail.org/)
+  if not promedFeed?.title
+    newFeedProps =
+      title: 'ProMED-mail'
+      url: 'promedmail.org/post/'
+    feedSchema.validate(newFeedProps)
+    Feeds.upsert promedFeed?._id,
```

Comment:
Seems like if the id is null, then a new document would be inserted since its an `upsert`?